result/isislsp.yaml

```diff
@@ -432,27 +461,326 @@ components:
+        sid:
+          description: >-
+            SID/Index is the SID/Label value associated 
+            with the IGP Prefix segment attached to the specific IPv4 or IPv6 prefix.
+          type: integer
+          format: uint32
+          x-field-uid: 1
+        flags:
           description: |-
-            External Prefix Flag (Bit 0)
+            Flags associated with Prefix Segment-ID.
+          type: array
+          items:
+            $ref: '#/components/schemas/IsisLsp.PrefixSidOneFlag'
+          x-field-uid: 2
+        algorithm:
+          description: |-
+            The Isis may use various algorithms when calculating
+            reachability to other nodes or to prefixes attached to these
+            nodes. 
+          type: integer
+          format: uint32
+          minimum: 0
+          maximum: 255
+          x-field-uid: 3
+
+    IsisLsp.Capability:
+      description: >-
+        Container of IS-IS Router CAPABILITY TLV.
+      type: object
+      properties:
+        router_cap_id:
+          description: >-
+            Router CapabilityID in IPv4 address format.
+          type: string
+          format: ipv4
+          x-field-uid: 1
+        flood:
+          description: |-
+            For S bit (0x01): If the S bit is set(1), the IS-IS Router CAPABILITY TLV
+            MUST be flooded across the entire routing domain.  If the S bit is
+            not set(0), the TLV MUST NOT be leaked between levels.  This bit MUST
+            NOT be altered during the TLV leaking.
+          type: boolean
+          x-field-uid: 2
+        down:
+          description: |-
+            For D bit (0x02): When the IS-IS Router CAPABILITY TLV is leaked from
+            Level 2 (L2) to Level 1 (L1), the D bit MUST be set.  Otherwise, this
+            bit MUST be clear.  IS-IS Router CAPABILITY TLVs with the D bit set
+            MUST NOT be leaked from Level 1 to Level 2.  This is to prevent TLV looping.
+          type: boolean
+          x-field-uid: 3
+        sr_capability:
+          description: |-
+            SR-Capabilities.      
+          $ref: '#/components/schemas/IsisLsp.SRCapability'
+          x-field-uid: 4     
+        algorithms:
+          description: |-
+            This contains one or more SR-Algorithm.
+          type: array
+          items:
+            $ref: '#/components/schemas/IsisLsp.Algorithm'
+          x-field-uid: 5
+        srlbs:
+          description: |-
+            This contains the list of SR Local Block (SRLB)
+          type: array
+          items:
+            $ref: '#/components/schemas/IsisLsp.Srlb'
+          x-field-uid: 6
+
+    IsisLsp.SRCapability:
+      description: |-
+        Container of IS-IS SR-CAPABILITY Sub-Tlv.
+      type: object
+      properties:
+        ipv4_mpls:
+          description: |-
+            MPLS IPv4 Flag. If set, then the router is capable of processing SR-MPLS-encapsulated IPv4 packets on all interfaces.
           type: boolean
```

Comment:
default ?

Reply:
Its Learned Info. does not need default value.